### ISSUE TITLE
Add the ability to crop profile images or set a fixed "square" dimensions image upload

### ISSUE THREAD

**Issue by Artmir** [linnovate/openideal#127]
I guess this is is not as much an issue as it is an enhancement suggestion. Currently when uploading user profile images, when viewed in front page they appear stretched on one side due to different height and width uploaded image. It would be great to add manual cropping possibility or limit profile image uploads to "square" dimensions only. I.e. (680x680)

**Comment by zohar** [comment]
Indeed, cropping of images would be good also for all content types. It is already in our roadmap, but not yet scheduled for development. 
PRs are welcome :-)
